examples/platform/posix/platform.h

```diff
@@ -36,34 +36,94 @@
 #define PLATFORM_H_
 
 #include <stdint.h>
+#include <sys/select.h>
+#include <sys/time.h>
 
 #ifdef __cplusplus
 extern "C" {
 #endif
 
 /**
+ * This method performs all platform-specific initialization.
+ *
+ */
+void PlatformInit(void);
+
+/**
+ * This method performs all platform-specific processing.
+ *
+ */
+void PlatformProcessDrivers(void);
+
+/**
  * This method initializes the alarm service used by OpenThread.
  *
  */
-void hwAlarmInit(void);
+void PlatformAlarmInit(void);
```

Comment:
should AlarmInit be part of platform API? It looks like this is an internal call only, right?